wren.go

```diff
@@ -318,6 +344,51 @@ func write(vm *C.WrenVM, text *C.char) {
 	fmt.Fprint(out, C.GoString(text))
 }
 
+//helper
+func readModule(dir string, name string) (string, error) {
+	var err error
+
+	// Precedence (dir/name.wren) next (dir/name/module.wren)
+	filename := filepath.Join(dir, name+".wren")
+	if _, err = os.Stat(filename); os.IsNotExist(err) {
+		filename = filepath.Join(dir, name, "module.wren")
+		if _, err = os.Stat(filename); os.IsNotExist(err) {
+			return "", err
+		}
+	}
+	var data []byte
+	if data, err = ioutil.ReadFile(filename); err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+//export loadModule
+func loadModule(vm *C.WrenVM, name *C.char) *C.char {
+	var module string = C.GoString(name)
+	var source string
+
+	// Ensure module does not have undesired characters
+	// that can pose thread to remote-code-inclusions
+	if !strings.Contains(module, "..") {
```

Comment:
One good trick for reducing indentation in a method is to return early, which I think would make sense here:

```go
if strings.Contains(module, "..") {
    return C.CString(source) // or nil, if it makes sense to
}
// rest of code
return C.CString(source)
```

Reply:
Adopted early return pattern now.